.github/workflows/tests.yml

```diff
@@ -55,11 +55,8 @@ jobs:
           ]
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '18'
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v2.4.2
+      - uses: actions/cache@v2.1.4
```

Comment:
(non-blocking nitpick) Same-I'd prefer these to require v2 instead of specific versions.

Reply:
Made the suggested changes.